rotkehlchen/api/v1/fields.py

```diff
@@ -66,6 +67,56 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
+class IncludeExcludeListField(fields.Field):
+    """ A field that accepts an object like the following and deserializes it to the proper types.
+    {
+        "values": [val1, val2, val3],
+        "behaviour": "exclude"
```

Comment:
Add the proper literral directly here

```suggestion
        "behaviour": "NOT IN"
```